src/env.cc

```diff
@@ -450,6 +459,59 @@ void Environment::RunAndClearNativeImmediates() {
 }
 
 
+void Environment::ScheduleTimer(int64_t duration) {
```

Comment:
Can we add the unit of measurement to `duration`? E.g. `duration_ms` if it’s milliseconds?